e2e/volume_ops_test.go

```diff
@@ -383,9 +383,7 @@ func testVolumeInfo(t *testing.T) {
 }
 
 func testVolumeStatus(t *testing.T) {
-	if _, err := os.Lstat("/dev/fuse"); os.IsNotExist(err) {
-		t.Skip("skipping mount /dev/fuse unavailable")
-	}
+	checkFuseAvailable(t)
```

Comment:
I cant find this function. Am i missing anything?

Reply:
https://github.com/gluster/glusterd2/blob/eb548f7cf915b7528cd0605e0f744b3d796782d3/e2e/utils_test.go#L361